src/shared/components/search.tsx

```diff
@@ -1097,10 +1103,6 @@ export class Search extends Component<any, SearchState> {
       sort: sort ?? urlSort,
     };
 
-    this.props.history.push(`/search${getQueryString(queryParams)}`, {
-      searched: true,
-    });
-
-    await this.search();
+    this.props.history.push(`/search${getQueryString(queryParams)}`);
```

Comment:
Should we create a global `siteUrl` constant at some point, in case someone installs Lemmy in a subdirectory or something?